sbomdiff/spdx_parser.py

```diff
@@ -34,16 +34,31 @@ def parse_spdx_tag(self, sbom_file):
         packages = {}
         package = ""
         version = None
+        githubStr = "pkg.go.dev/"
```

Comment:
This is not appropriate as this only works for Go packages. sbomdiff needs to be generic for all SBOMs and languages

Reply:
Agree I will add some check to do this only for GO libs.

Reply:
Could you suggest how to find packages for other language like java, python etc.

Reply:
> Could you suggest how to find packages for other language like java, python etc.

I don't think this is a valid change, 

The differences in package name is due to the differences in the SBOM generator. Comparing SBOMs from different generators is a valid use case and I can see how useful it is to detect that the generators are creating different names for the same package. Trying to make sbomdiff cater for different generators and establish that the different package names are actually the same package is beyond the scope of sbomdiff as it is not viable to accommodate the approaches adopted by all the different SBOM generators for generating package names.